lib/setup.rb

```diff
@@ -47,7 +47,7 @@ def register_runner
     def write_token(token)
       puts "Runner token: #{token}"
 
-      Config.new.write('token', token)
+      Config.new(0).write('token', token)
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.